.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,66 +24,62 @@
+- [ ] I have added JavaScript unit tests to cover my changes
 - [ ] All new and existing tests passed
 
-<!------------------------->
-<!-- /!\ Core Team Only -->
-<!------------------------->
+#### Documentation
 
-<!-- Uncomment the following for a feature DoD -->
+- [ ] My change introduces changes to the documentation and/or I have updated the documentation accordingly
 
-<!--
-### Development
-
-- [ ] Should match specs (eg. either the Web UI Kit or any pattern from the WAI — or both…)
-- [ ] Docs added:
-  - including the "Sass" part using `scss-docs` shortcode
-  - in /about/overview/#custom-components if it is a new Orange custom component
-  - in /getting-started/introduction/#components if it is a new Orange custom component that requires JavaScript (and Popper)
-  - in /customize/overview#csps-and-embedded-svgs if it is a new Orange custom component that includes embedded SVGs in our CSS
-  - in /forms/validation/?#supported-elements if it is a new Orange custom component that is a form control
-  - in /forms/overview/ if it is a new Orange custom component that is a form control
-- [ ] (Optional) If this feature is a new component, it probably must appear in Storybook
-- [ ] Check (and fix) RTL version
-- [ ] Run linters
-- [ ] Run compilers
-- [ ] Tests added for JS-side
-- [ ] Run tests
-- [ ] Manually run BrowserStack test
-- [ ] Manually run Percy test
-- [ ] Cross-browser test:
-  - [ ] Chrome
-  - [ ] Firefox (including ESR)
-  - [ ] Edge
-  - [ ] Safari
-  - [ ] iOS Safari
-  - [ ] Chrome & Firefox on Android
-- [ ] Responsive tests: desktop and small screens
-- [ ] Clean up the branch using `rebase -i`
-- [ ] Commited with `feat(…): …` message
-- [ ] Mention it in Migration Guide (if `back-from-v4`): renamed variables, changes in markup requirement,  etc.
-
-### Reviews
+<details>
+  <summary>Detailed info</summary>
+  <ul>
+    <li>including the "Sass" part using <code>scss-docs</code> shortcode</li>
+    <li>in <code>/about/overview/#custom-components</code> if it is a new Orange custom component</li>
+    <li>in <code>/getting-started/introduction/#components</code> if it is a new Orange custom component that requires JavaScript (and Popper)</li>
+    <li>in <code>/customize/overview#csps-and-embedded-svgs</code> if it is a new Orange custom component that includes embedded SVGs in our CSS</li>
+    <li>in <code>/forms/validation/?#supported-elements</code> if it is a new Orange custom component that is a form control</li>
+    <li>in <code>/forms/overview/</code> if it is a new Orange custom component that is a form control</li>
+  </ul>
+</details>
+
+### Checklist (for Core Team only)
 
+- [ ] My change introduces changes to the migration guide
+- [ ] My new component is added in Storybook
+- [ ] My new component is compatible with RTL
+- [ ] Manually run [BrowserStack automatic tests](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/actions/workflows/browserstack.yml)
```

Comment:
Should we add this change to the DoD for release as well ? (haven't checked all the changes)

Reply:
For the DoD for release, we won't do more tests since it is already done before for each pull request. IMO the release DoD will only contain steps to actually do the release (without testings).

Reply:
You mean updating the DoD for release? Can do it in another PR so we focus on one thing at the time.